control_plane/src/lib.rs

```diff
@@ -34,31 +33,41 @@ pub fn read_pidfile(pidfile: &Path) -> Result<i32> {
     Ok(pid)
 }
 
-fn fill_rust_env_vars(cmd: &mut Command) -> &mut Command {
-    let cmd = cmd.env_clear().env("RUST_BACKTRACE", "1");
+// We have `cmd: &mut Command` and `cmd: subprocess::Exec` arguments possible here
+// that have the same API in terms of manipulating the env vars
+// use the same AST for that on different types
+#[macro_export]
+macro_rules! fill_rust_env_vars {
```

Comment:
The old function version was much easier to understand. With the macro, it's not clear what the argument is and what this does to it. Needs a comment with usage examples, at least.

Reply:
Both places now use `subprocess::Exec` hence macro is not needed anymore, reverted back to function.